static/public/index.html

```diff
@@ -6,6 +6,359 @@
+              <label for="text" class="w-7 h-7 text-grey static text-xs">Default</label>
+              <div class="input-container">
+                <input type="text" placeholder="Enter text" class="input-field"/>
+              </div>
+            </div>
+            <div class="flex flex-col">
+              <div class="w-80 h-10 flex-col justify-start">
+                <label for="text" class="w-7 h-7 text-grey static text-xs">focus</label>
+                <div class="input-container-focus">
+                  <input type="text" placeholder="Enter text" class="input-field"/>
+                  <svg class="input-icon" width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M7.00016 0.333313C3.32416 0.333313 0.333496 3.32398 0.333496 6.99998C0.333496 10.676 3.32416 13.6666 7.00016 13.6666C10.6762 13.6666 13.6668 10.676 13.6668 6.99998C13.6668 3.32398 10.6762 0.333313 7.00016 0.333313ZM9.80483 8.86198L8.86216 9.80465L7.00016 7.94265L5.13816 9.80465L4.1955 8.86198L6.0575 6.99998L4.1955 5.13798L5.13816 4.19531L7.00016 6.05731L8.86216 4.19531L9.80483 5.13798L7.94283 6.99998L9.80483 8.86198Z" fill="#8D9A9E"/>
+                    </svg>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="flex flex-col md:flex-row gap-16">
+            <div class="flex flex-col">
+              <div class="w-80 h-10 flex-col justify-start">
+                <label for="text" class="w-7 h-7 text-grey static text-xs">Error</label>
+                <div class="input-container-error">
+                  <input class="input-field" type="text" placeholder="Enter text">
+                  <svg class="input-icon" width="14" height="14" viewBox="0 0 2 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M0.333496 0.666626H1.66683V5.33329H0.333496V0.666626ZM0.333496 5.99996H1.66683V7.33329H0.333496V5.99996Z" fill="#FE544A"/>
+                    </svg>                
+                </div>
+              </div>
+            </div>
+            <div class="flex flex-col">
+              <div class="w-80 h-10 flex-col justify-start">
+                <label for="text" class="w-7 h-7 text-grey static text-xs">Success</label>
+                <div class="input-container-success">
+                  <input class="input-field" type="text" placeholder="Enter text">
+                  <svg width="11" height="9" viewBox="0 0 11 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M3.66681 6.39061L1.47148 4.19527L0.528809 5.13794L3.66681 8.27594L10.1381 1.8046L9.19547 0.861938L3.66681 6.39061Z" fill="#07B255"/>
+                    </svg>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="flex flex-col">
+            <div class="flex flex-row justify-start align-middle gap-32">
+
+              <label class="switchBtn align-middle">
```

Comment:
- The **hover** effect required on the **background** and **focus** required on **border**.
- take a look on [tailwind hover](https://tailwindcss.com/docs/hover-focus-and-other-states).
- try to understand the idea behind the focus on this [codepen](https://codepen.io/losbeekos/pen/zsdFa?editors=1100) it may help you.